ui/util/api/endpoints.js

```diff
@@ -14,7 +14,8 @@ const CACHE_CONFIG = {
 
 export class Endpoint {
   constructor(endpoint) {
-    this.client = axios.create({ baseURL: process.env.API_URL });
+    const baseURL = process.env.INTERNAL_API_URL || process.env.API_URL;
```

Comment:
I could add some logic here that only uses `INTERNAL_API_URL` if `process.browser` is falsy; while it isn't strictly required to make the code work--`process.env.INTERNAL_API_URL` isn't passed to webpack's environment plugin and therefore will always be falsy when run on the browser-side--it could be helpful in communicating intent to readers.

Reply:
That's a smart idea; it is rather cryptic now.